.github/workflows/test.yml

```diff
@@ -68,9 +69,14 @@ jobs:
       - name: Build routes
         run: pnpm build
       - name: Test all and generate coverage
-        run: pnpm run vitest:coverage
+        run: pnpm run vitest:coverage --outputFile=tmp/vitest-coverage.xml
```

Comment:
This breaks 
https://github.com/DIYgod/RSSHub/blob/044d17fb3366ed9e01e244d1b0a175987c08587b/.github/workflows/test.yml#L80-L84
since it can no longer read the HTML generated from coverage folder.

Reply:
What do you mean by “HTML generated from coverage folder”? When I run the `vitest:coverage` script (without the flag), it doesn’t generate any HTML in that folder either, only a bunch of JSON files like `coverage/.tmp/coverage-x.json`. Were you referring to those JSON files?

Reply:
Please run the test with `GITHUB_ACTIONS=1`.

Reply:
I searched `GITHUB_ACTIONS` in Vitest documentation, and found [GitHub action reporter](https://vitest.dev/guide/reporters.html#github-actions-reporter). I think it is a better solution than the solution I proposed, for two reasons:

1. We don’t need to introduce a new dependency.
2. Less likely to break the Codecov job.

I updated the PR title to reflect that. I assumed that we squash commits when merging, so I pushed a new commit instead of amending the existing one.